internal/routing/local.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/qdm12/gluetun/internal/netlink"
 )
 
+const (
+	mainTable     = 254
+	localPriority = 98
```

Comment:
Can you perhaps add a tiny comment on why these values please 🙏

Reply:
Added comment on why these value and moved to be locals.  Please LMK what you think.